bats/helpers/funding/wallet.bash

```diff
@@ -0,0 +1,11 @@
+balance_for_wallet() {
```

Comment:
not sure about having this under `funding`.
Funding (for me) implies sending / seeding money into a wallet. Getting a balance is a more general function.

What about something like this:
```
helpers
- onchain.bash
- intraledeger.bash
- ln.bash
- wallet.bash
```

Or something like that.

Reply:
fixed in 9757bd4e56a6b4117e750b1782765b5ef35556e2